packages/esm-appointments-app/src/patient-appointments/patient-appointments-overview.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { usePatient, useLayoutType } from '@openmrs/esm-framework';
+import PatientAppointmentsBase from './patient-appointments-base.component';
+import { useParams } from 'react-router-dom';
+import Overlay from '../overlay.component';
+import PatientAppointmentContext, { PatientAppointmentContextTypes } from '../hooks/patientAppointmentContext';
+import PatientAppointmentsHeader from './patient-appointments-header';
+import { DataTableSkeleton } from '@carbon/react';
+
+const PatientAppointmentsOverview: React.FC = () => {
+  let params = useParams();
+  const response = usePatient(params.patientUuid);
+  const layout = useLayoutType();
+  const isDesktop = layout === 'small-desktop' || layout === 'large-desktop';
```

Comment:
`esm-framework` exports an `isDesktop` function that accepts a `layout` argument and returns the appropriate boolean value https://github.com/openmrs/openmrs-esm-core/blob/main/packages/framework/esm-framework/docs/API.md#isdesktop.